src/MyHealth/OverTime.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from "react"
+import { View } from "react-native"
+import { useTranslation } from "react-i18next"
+
+import { useSymptomLogContext } from "./SymptomLogContext"
+import { HealthAssessment } from "./symptoms"
+import { GlobalText } from "../components"
+
+const OverTime: FunctionComponent = () => {
+  const { t } = useTranslation()
+  const { logEntries } = useSymptomLogContext()
+
+  return (
+    <View>
+      {logEntries.map((logEntry) => {
+        const healthAssessmentText =
+          logEntry.healthAssessment === HealthAssessment.AtRisk
+            ? t("my_health.symptom_log.feeling_not_well")
```

Comment:
If this is an intermediate bit of text, that is not intended for the final release, it would probably be better to not have it go through the translation process, so we could consider inline the string here.